src/configuration.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The RLS Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+'use strict';
+
+import { workspace, OutputChannel, WorkspaceConfiguration } from 'vscode';
+import { RevealOutputChannelOn } from 'vscode-languageclient';
+
+export { RevealOutputChannelOn } from 'vscode-languageclient';
```

Comment:
It's not strictly required, but since it consumes `RevealOutputChannelOn` from somewhere else and exports a function that contains this type, I thought it's good style to also export what's in the interface; I can remove it since it's only used locally in practice